src/components/ReportActionItem/MoneyRequestReportPreview/MoneyRequestReportPreviewContent.tsx

```diff
@@ -570,6 +579,14 @@ function MoneyRequestReportPreviewContent({
         ),
     };
 
+    const adjustScroll = useCallback(() => {
```

Comment:
Can you please add a comment explaining why we do this? Also reference that this is a workaround to solve an issue in React repo, also linking that issue

Reply:
@MonilBhavsar done